apps/voting/app/src/components/VotingCard/VotingCard.js

```diff
@@ -41,7 +41,7 @@ class VotingCard extends React.Component {
         </Header>
         <Card>
           <Content>
-            <Question>
+            <Question title={`ID: ${id}`}>
```

Comment:
One last thing (haha it never ends sorry 😄): the `title` attribute takes precedence when being read by web browsers / screen readers, that’s why it should always reflect the full information it represents (ID + title in that case).

Reply:
I did not consider showing the title itself, so, sure using: `I am a title (ID: 21)` works for me.

Reply:
@bpierre had to go fetch the `description` one component up.

Reply:
It shouldn’t be needed, we already have `question` here! :-)

```suggestion
            <Question title={`${question} (ID: ${id})`}>
```

Reply:
`question` is a React component :( that's the first thing I tried, but it has 3 spans.

Reply:
Oooh sorry 🤦‍♂️ I understand now.